cob_obstacle_distance_moveit/src/cob_obstacle_distance_moveit/obstacle_distance_moveit.cpp

```diff
@@ -169,13 +160,14 @@ void ObstacleDistanceMoveit::calculateDistanceTimerCallback(const ros::TimerEven
         {
             std::string robot_self_name = selfcollision_it->first;
             collision_detection::AllowedCollision::Type type;
-            if(acm.getEntry(robot_link_name, robot_self_name, type))
+
+            if(acm_.getEntry(robot_link_name, robot_self_name, type))
             {
-                if(type == collision_detection::AllowedCollision::NEVER)
+                if(type == collision_detection::AllowedCollision::ALWAYS)
```

Comment:
`NEVER`: the tests were not able o finf collisions, so it is assumed that they will never happen
` ALWAYS`: There have been so many collisions that you can barely move if you don't ignore them. (it is a superset of `DEFAULT`)
These additions are just meant as annotation to understand why these pairs got disabled.

In all cases the collision detector does not run checks for these pairs.
`ALWAYS` pairs should be taken with care, so don't add them if it could damage something.